packages/forma-36-website/src/components/Layout.css

```diff
@@ -1,4 +1,4 @@
-@import './../../node_modules/@contentful/forma-36-tokens/dist/css/index.css';
+@import '~@contentful/forma-36-tokens/dist/css/index.css';
```

Comment:
I guess no one else actually uses this package, but in theory, if they did, would this import work for them? Shouldn't it be absolute?

Reply:
unfortunately, that works with webpack, we're using `node-sass` directly. we should revisit the build system for `fcss` packages later, or even deprecate it, I remember Johannes mentions multiple times it's not being used much.

Reply:
Uhmm I don't know now
`@contentful/forma-36-tokens` is a dependency in this package (website)
and I thought that `~` just tells webpack (well gatsby using webpack) to get this in node_modules
isn't it how this works?

Reply:
Ah yeah, it's the website, it should be fine. I thought it's the `fcss` package, sorry 🤦

Reply:
@mshaaban0 your point about the `fcss` is still a valid point tough
I think `tokens` pkg is using it, but maybe we could avoid it